musica-fortran/src/micm/micm_mod.F90

```diff
@@ -0,0 +1,63 @@
+module micm
+    use iso_c_binding
+
+    private
+    public :: micm_t
+
+    include "micm_c_def.F90"
+
+    type micm_t
+        private
+        type(c_ptr) :: ptr
+    contains
+        procedure :: delete => delete_micm
+        procedure :: create_solver => micm_create_solver
+        procedure :: solve => micm_solve
+    end type
+
+    interface micm_t
+        procedure create_micm
+    end interface
+
+contains
+    function create_micm(config_path)
+        implicit none
```

Comment:
Please place `implicit none` at the top of this on the line before `private` and remove `implicit none` in all of these functions

Reply:
Yes, it's done now.